tests/framework/extensions/workloads/create.go

```diff
@@ -0,0 +1,69 @@
+package workloads
+
+import (
+	stevev1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+	"github.com/rancher/rancher/tests/framework/extensions/services"
+	"github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	containerNamePrefix = "tcon"
+	wlNamePrefix        = "twl"
+	defaultNamespace    = "default"
+	isCattleLabeled     = true
+	serviceNamePrefix   = "svc-"
+)
+
+func CreatSimpleDeployment(steveclient *stevev1.Client, containerImage string) (*stevev1.SteveAPIObject, error) {
```

Comment:
I think we can call this CreateDeployment, although I don't have a strong opinion about having a create call. Before the Steve client, we needed them because we wanted to wrap unstructured -> k8s object type changes.

Right now we mainly rely on templates and make the create call plainly inside the functions, please check my comment on create service

Reply:
this along with the service was reverted as discussed offline.